marketdata/stream/client.go

```diff
@@ -216,6 +221,24 @@ func (cc *CryptoClient) constructURL() (url.URL, error) {
 	return constructURL(cc.baseURL, cc.feed)
 }
 
+// This will create a crytpo perpetual futures subscriptions to the Alpaca market data services
+type CryptoPerpsClient struct {
```

Comment:
I don't see a benefit of adding this for now. The `CryptoClient` can be used instead. If we add a new message type to the perpetual future stream, then a new client will be needed, but also it will require additional changes.

Reply:
Yes, for now there is no benefit. But we added `CryptoPerpsClient` because we assumed that in the future it will be needed for the `futures-pricing-data`.